syz-fuzzer/proc.go

```diff
@@ -42,6 +42,8 @@ func newProc(fuzzer *Fuzzer, pid int) (*Proc, error) {
 	rnd := rand.New(rand.NewSource(time.Now().UnixNano() + int64(pid)*1e12))
 	execOptsCollide := *fuzzer.execOpts
 	execOptsCollide.Flags &= ^ipc.FlagCollectSignal
+	// We make some calls async, so it's reasonable to also decrease the overall timeout.
```

Comment:
For completeness we also need to reflect this in pkg/csource, it also uses timeouts during program generation. This one may be not so important, but it's bad if they diverge overall (there are more important timeouts).
I wonder if there is a way to not duplicate this logic here and in pkg/csource... but not sure where else we could put this logic.

Reply:
Maybe it's easier to just drop that commit - to be stopped via the `Program` timeout is a pretty rare event given the per-syscall timeouts.

Reply:
You know better.